src/views/Simulation/utils/pixiUtils.js

```diff
@@ -134,13 +134,13 @@ const createFactoryIconGraphicsContext = (options) => {
   return graphicsContext;
 };
 
-const createStockContainer = (textures, name, isHighlighted = false) => {
+const createStockContainer = (textures, name, isHighlighted = false, settings) => {
   const container = new Container();
   container.label = isHighlighted ? NODE_TYPES.STOCK_SHORTAGE : NODE_TYPES.STOCK;
   const stockTextureKey = isHighlighted ? 'stockLevel1' : 'stockLevel0';
   if (isHighlighted) {
     const stockHalo = new Sprite(textures[stockTextureKey]);
-    stockHalo.filters = [BLOOM_FILTER];
+    stockHalo.filters = settings.filterEffectActivated ? [BLOOM_FILTER] : [];
```

Comment:
Instead of providing the whole `settings` object, could we replace it by a simple boolean value `enableGlowEffect` to keep this function low-level and focused on the Pixi-related logic?
Same thing for the functions `createProductionResourceContainer` and `createNodeContainer` if it's possible